interp/declare.ml

```diff
@@ -153,7 +149,9 @@ let definition_entry ?fix_exn ?(opaque=false) ?(inline=false) ?types
     const_entry_feedback = None;
     const_entry_inline_code = inline}
 
-let declare_constant ?(internal = UserIndividualRequest) ?(local = false) id ?(export_seff=false) (cd, kind) =
+(* TODO: move me somewhere in the STM. *)
```

Comment:
A voice told me this is a @maximedenes plan, can you explain more? I have no clue what that would look like.

Reply:
@maximedenes that's your plan, I summon you to defend it. Otherwise I'll just remove the TODO.

Reply:
TBH I'm not sure of the value of leaving the TODO even with the plan defended. A TODO with local implications on the code OK sense, but if there's a need for a plan maybe it should live as an issue or cep.

Reply:
I'm afraid I can do much about hearing voices :)

Reply:
It was not a plan, just a potential idea. I agree that a TODO doesn't seem like the right tool here.

Reply:
The TODO has been removed anyways.

Reply:
Ah right. Please mark discussions as resolved when they are, it saves time for other reviewers.